include/RAJA/policy/sycl/forall.hpp

```diff
@@ -455,10 +416,8 @@ RAJA_INLINE resources::EventProxy<resources::Sycl> forall_impl(resources::Sycl &
                      loop_body);
   }  // iterate over segments of index set
 
-  if (!Async) {
-    ::sycl::queue* q = ::RAJA::sycl::detail::getQueue(); 
-    q->wait();
-  }
```

Comment:
Why was the check on Async removed?

Reply:
Thanks for catching that. It was removed unintentionally. I will re-add.